streaming/java/streaming-state/src/main/java/org/ray/streaming/state/serde/IStoreSerDe.java

```diff
@@ -0,0 +1,8 @@
+package org.ray.streaming.state.serde;
+
+/**
+ * serde interface.
```

Comment:
Please add meaningful document or just remove it...

I don't think other contributors could get anything from this document.

So, `IStoreSerde` is a tagged interface, what is the behavior interface/class with this interface should do? StoreSerde sounds like a byte array store, but it is really general.

Reply:
Thanks for your comment. IStoreSerde is useless indeed, it will be removed